easybuild/easyconfigs/a/ASE/ASE-3.13.0-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
+
+name = 'ASE'
+version = '3.13.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://wiki.fysik.dtu.dk/ase'
+description = """ASE is a python package providing an open source Atomic Simulation Environment
+ in the Python scripting language."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+checksums = ['e946a0addc5b61e5e2e75857e0f99b89']
+
+dependencies = [
+    ('Python', '2.7.12'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/ase-build', 'bin/ase-db', 'bin/ase-gui', 'bin/ase-info', 'bin/ase-run'],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/%(namelower)s'],
+}
+
+moduleclass = 'chem'
+
```

Comment:
@pouillon please remove this trailing blank line to make the automatic style tests pass, you can check yourself using `eb --check-style`